src/scss/settings/_component.scss

```diff
@@ -0,0 +1 @@
+$borderRadius: 6px;
```

Comment:
Instead `settings/_components.scss`, let's move to `settings/_borders`. `$borderRadius` does not appear to be exclusive of a component.